masterlist.yaml

```diff
@@ -25181,3 +25198,16 @@ plugins:
       - <<: *useOnlyOneX
         condition: 'many("(CFR-CloaksForRequiem|Requiem - Cloaks( of Skyrim| of Skyrim Hard - Times| of Skyrim Patch| WICCloaks CCFR USKP)?)\.esp")'
         subs: [ 'Requiem patch for Cloaks of Skyrim' ]
+  - name: 'Requiem - Immersive Jewelry( BTC| ISC)?\.esp'
```

Comment:
I think the "issue" is that `Requiem - Immersive Jewelry.esp` is also an existing file name, so if the space is to be separated it would become `Requiem - Immersive Jewelry( (BTC|ISC)?)\.esp` which is not really much more neat…